src/brogue/RogueMain.c

```diff
@@ -130,14 +130,34 @@ static void benchmark() {
     printf("\n\nBenchmark took a total of %lu seconds.", ((unsigned long) time(NULL)) - initialTime);
 }
 
+static const char *getOrdinalSuffix(int number) {
```

Comment:
I thought returning `char *` was wrong but string literals have static duration so this is legit!